projects/wvr-elements/src/lib/shared/animation/wvr-animations.ts

```diff
@@ -0,0 +1,146 @@
+import { animate, AnimationMetadata, style } from '@angular/animations';
+
+const wvrAnimationDefaults = {
```

Comment:
Do we want to consider externalizing here defaults? Seems to be useful to be able to pick different defaults,

Reply:
Thats a great idea.